Vagrantfile

```diff
@@ -78,13 +87,17 @@ cd /vagrant # This is where the host folder/repo is mounted
 gem install bundler foreman
 bundle install
 
-# Install node modules
+# Disable husky inside vagrant:
+echo "HUSKY=0" >> ~/.bash_profile
```

Comment:
If husky is enabled inside vagrant, it installs a bunch of additional `.husky/` scripts, which aren't present when running `yarn install` outside of the vagrant instance.